apps/client/src/pages/time-table/page/time-table-page.tsx

```diff
@@ -1,70 +1,41 @@
 import { useFestivalSelect } from '@pages/time-table/hooks/use-festival-select';
-import { useImageDownload } from '@pages/time-table/hooks/use-image-download';
-import { useTimeTableEdit } from '@pages/time-table/hooks/use-time-table-edit';
+import EmptyFestivalSection from '@pages/time-table/page/empty/empty-festival-section';
+
+import { SwitchCase } from '@shared/components/switch-case';
 
 import {
   useFestivalButtonData,
   useFestivalTimetableData,
+  useTimeTableCreationHistory,
 } from '../hooks/use-festival-data';
-
-import * as styles from './time-table-page.css';
+import TimeTableLoaded from './loaded/time-table-loaded';
+import TimeTableOnboard from './onboading/time-table-onboard-page';
 
 const TimeTablePage = () => {
-  const { isEditTimeTableMode, toggleEditTimeTableMode } = useTimeTableEdit();
   const { festivals } = useFestivalButtonData();
-
-  const {
-    selectedFestivalInfo,
-    selectedDateId,
-    handleSelectFestival,
-    handleSelectDate,
-  } = useFestivalSelect(festivals);
-
+  const { hasTimetableHistory } = useTimeTableCreationHistory();
+  const { selectedDateId } = useFestivalSelect(festivals);
   const { data: boardData } = useFestivalTimetableData(selectedDateId);
 
-  const { elementRef, downloadImage } = useImageDownload<HTMLDivElement>({
-    fileName: `${selectedFestivalInfo.title}`,
-  });
+  const timetableState = useMemo<'onboard' | 'empty' | 'loaded'>(() => {
+    if (!hasTimetableHistory) return 'onboard';
+    if (festivals.length === 0) return 'empty';
+    if (festivals.length > 0 && boardData) return 'loaded';
+    return 'empty';
+  }, [hasTimetableHistory, festivals, boardData]);
 
   return (
-    <>
-      {festivals.length === 0 ? (
-        <EmptyFestivalSection />
-      ) : (
-        <div className={styles.wrapper}>
-          <FestivalSelector
-            festivals={festivals}
-            selectedFestivalId={selectedFestivalInfo.festivalId}
-            handleSelectFestival={handleSelectFestival}
-          />
-          <Calender
-            festivalDates={selectedFestivalInfo.festivalDates}
-            onDateSelect={handleSelectDate}
-          />
-          {boardData && (
-            <div className={styles.timeTableWrapper} ref={elementRef}>
-              <FestivalStage timeTableInfo={boardData} />
-              <TimeTableBoard
-                timeTableInfo={boardData}
-                isEditMode={isEditTimeTableMode}
-              />
-            </div>
-          )}
-
-          <TimeTableActions
-            isEditMode={isEditTimeTableMode}
-            onToggleEditMode={toggleEditTimeTableMode}
-            onDownload={downloadImage}
-          />
-        </div>
-      )}
-    </>
+    <SwitchCase
+      value={timetableState}
+      caseBy={{
+        onboard: () => <TimeTableOnboard />,
+        empty: () => <EmptyFestivalSection />,
+        loaded: () => (
+          <TimeTableLoaded festivals={festivals} boardData={boardData} />
```

Comment:
여기두,,컴포넌트 명 TimeTableContent가 더 명확할 것 같은데 어떻게 생각하시나요 ㅎㅎ
```suggestion
        render: () => (
          <TimeTableContent festivals={festivals} boardData={boardData} />
```

Reply:
오케이~ 반영할게요~